test/myFTClient.spec.js

```diff
@@ -913,6 +914,28 @@ describe('myFT Client proxy', () => {
 					.catch(done);
 			});
 
+			it('Should get an array of concepts followed by a group', done => {
+				if (mockAPI) {
```

Comment:
Shouldn't need to use the mockAPI flag, so just see if it all still builds when you take it out.

Reply:
should be please to know mockAPI is gone. At least from the myftClient.spec.js